src/components/TextArea/AutoFocusTextArea.js

```diff
@@ -46,7 +47,7 @@ AutoFocusTextArea.propTypes = {
 
 const styles = StyleSheet.create({
   input: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the input field has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that `COLORS.gray100` is defined and provides the desired color for your UI. If it's not defined, you might face an undefined variable error.
```diff
-    backgroundColor: COLORS.gray100,
+    backgroundColor: COLORS.gray100 || COLORS.lightgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->